TASK-5/Customer.cs

```diff
@@ -0,0 +1,32 @@
+namespace Task5;
+
+public class Customer
+{
+    private string _passportId;
+    private string _name;
+    private int _age;
+    private Cart _cart;
+
+    public string PassportId { get => _passportId; set => _passportId = value; }
+    public string Name { get => _name; set => _name = value; } 
+    public int Age { get => _age; set => _age = value; }
+    public Cart Cart { get => _cart; set => _cart = value; }
```

Comment:
Better to write { get ; set; } for fields, for example:
public string _passportId { get; set; }
Please, Change it everywhere

Reply:
Done.